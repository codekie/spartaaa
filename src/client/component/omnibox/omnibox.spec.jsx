import React from 'react';
import Omnibox from './omnibox.jsx';
import { shallow } from 'enzyme';

const VAL = '';

it('renders correctly', () => {
    const parsedValues = {
        project: 'foo',
        tags: ['bar']
    };
    expect(shallow(<Omnibox rawValue={VAL} parsedValues={parsedValues} />)).toMatchSnapshot();
});
