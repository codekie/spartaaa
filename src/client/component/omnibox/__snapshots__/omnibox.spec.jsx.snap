// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Input
  className="cmp-sub-input"
  color={null}
  disabled={false}
  isStatic={false}
  name={null}
  onChange={[Function]}
  onKeyDown={[Function]}
  placeholder="search"
  readOnly={false}
  size="small"
  style={Object {}}
  type="text"
  value=""
/>
`;
